index.html

```diff
@@ -123,6 +123,71 @@
         </div>
       </ul>
     </section>
+    <section class="Earnings__right-side" aria-label="Here you can enter all the requested capes">
+      <img class="Earnings__right-sign" src="image/Frame.png" alt="logo Ember">
+      <dl class="Earnings__right-sign-up">
+        <dt class="Earnings__right-title">Sign up for an account</dt>
+        <dd class="Earnings__right-text">Send, spend and save smarter</dd>
+      </dl>
+      <div class="Earnings__right-button">
+        <button class="Earnings__right-google" type="button" aria-label="Sign up with Google">
+          <img class="Earnings__right-img" src="image/Google.png" alt="Google Logo">Sign Up with Google </button>
+        <button class="Earnings__right-apple" type="button" aria-label="Sign up with Apple">
+          <img class="Earnings__right-img" src="image/Apple logo.png" alt="Apple Logo">Sign Up with Apple </button>
+      </div>
+      <div class="Earnings__right-function">
+        <div class="Earnings__right-bord"></div>
+        <p class="Earnings__right-option" aria-label="login or with google">or with e-mail</p>
+        <div class="Earnings__right-bord"></div>
+      </div>
+      <form class="Earnings__right-form" id="js-sign-up-form" aria-label="fill all empty hoods">
```

Comment:
Las clases `js-` son _clases_, no _id_
```suggestion
      <form class="Earnings__right-form js-sign-up-form" aria-label="sign up form">
```